### DISCUSSION TITLE
use trained model for new project with similar search strategy

### DISCUSSION THREAD

**Discussion by IcexTrice** [asreview/asreview#1454] [Q&A] [answered]
Is there any built-in feature allowing us to use the same model for the next project with similar articles retrieved from a different library and/or load several libraries into one project? Would be very useful / highly appreciated when doing reviews with multiple search engines

**Comment by Rensvandeschoot** [comment]
In every iteration, a new model is trained using the available labeled records; see for a detailed description in the pre-print article [Reproducibility and Data Storage for Active Learning-Aided Systematic Reviews](https://psyarxiv.com/g93zf). 

The model parameters are stored in the `.asereview` project file; see how to [access data from the project file](https://asreview.readthedocs.io/en/latest/example_api_asreview_file.html).

**Comment by rohitgarud** [comment, marked as answer]
Hi @IcexTrice,
you can combine datasets from different libraries using the compose function from [ASReview-datatools](https://github.com/asreview/asreview-datatools).
If you add the new data below the exported dataset (CSV file) from the previous project which has labels, it will be used as prior for the new project and you will get the best classifier from the last project (it will train new model on the labeled records). However, this will only be the right thing to do if the inclusion, exclusion, and scope of the new project are the same as the previous one.
